SensorController/Core/Src/main.c

```diff
@@ -190,8 +200,9 @@ void SystemClock_Config(void)
   /** Initializes the RCC Oscillators according to the specified parameters
   * in the RCC_OscInitTypeDef structure.
   */
-  RCC_OscInitStruct.OscillatorType = RCC_OSCILLATORTYPE_HSE;
+  RCC_OscInitStruct.OscillatorType = RCC_OSCILLATORTYPE_LSI|RCC_OSCILLATORTYPE_HSE;
```

Comment:
Is this for the IWDG timer/clk?

Reply:
I didn't touch the clock code, so most likely yes. The only changes I made to main after code generation was adding the init function, LED toggle, and removing generated functions